building/windows.md

```diff
@@ -0,0 +1,240 @@
+# Windows
+
+For now, it is possible to build Phoenix-RTOS images for all targets and run `ia32-generic-qemu` on Windows using
+`Cygwin` or `WSL`. Running the rest of targets is under development.
+
+## Cygwin
+
+To start, we need to download and install the QEMU environment on Windows. It's important to do it first to avoid issues
+with the `qemu-integration` tool.
+
+To do this, go to the official [QEMU website](https://www.qemu.org/download/#windows) and select a mirror targeted
+for your system.
+
+![OS bit selection](_images/qemu-os-select.png)
+Then it should be installed with the default configuration.
+
+Next up we need to download `Cygwin` from the [official website](https://cygwin.com/install.html). It is a collection of
+Open Source tools that provides functionality similar to a Linux distribution on Windows.
+
+<details>
+<summary> Cygwin installation/update process (click to expand)</summary>
+&nbsp;
+
+* The installation process also serves as a package manager. `Cygwin` will ask about a type of installation in our case
+it will be first choice (Install from Internet).
+
+![cygwin-download-source](_images/cygwin-download-source.png)
+
+* After that, it will ask about the destination of a root and local package directory
+
+![cygwin-root-dir](_images/cygwin-root-dir.png)
+![cygwin-local-directory](_images/cygwin-local-directory.png)
+
+* `Cygwin` has the capability of using host's internet connection, but it needs to be configured with our preferences
+(In most scenario it will be System Proxy Settings)
+
+![cygwin-internet-connection](_images/cygwin-internet-connection.png)
+
+* Selection of a mirror for package downloads can be tricky, sometimes it needs to be changed for better results because
+not all mirrors have all available packets. That's why it's recommended to install only the necessary ones at first as
+it is done in the next instructions.
+
+![cygwin-mirror-selection](_images/cygwin-mirror-selection.png)
+
+* After selecting the mirror, we can select the packages we want. For now only `lynx` and `wget` should be chosen (they
+will be installed with required dependencies). Those tools are mandatory to use package manager
+`apt-cyg` and install the rest of required packages in Cygwin terminal (explained further).
+* Note that `Install` will be printed instead of `Reinstall` in case that you are executing the installation for a first
+time.
+
+![cygwin-selecting-packages](_images/cygwin-selecting-packages.png)
+
+* Once these are chosen, proceed to run the installation process.
+
+</details>
+  &nbsp;
+
+The next step is to run `Cygwin` terminal.
+Always run it as administrator, the best way to do that is simply setup this as default action:
+
+<details>
+<summary>How to run Cygwin as administrator by default (click to expand)</summary>

```

Comment:
I would look better with a newline as in the other places.

![Screenshot from 2023-11-10 11-44-39](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/77304431/daebe225-4584-431c-986d-b65fe0c65f60)

```suggestion
<summary>How to run Cygwin as administrator by default (click to expand)</summary>
&nbsp;
```